.gitignore

```diff
@@ -14,4 +14,4 @@ hs_err_pid*
 .idea/
 *.iml
 .gradle/
-
+build/
```

Comment:
I don't think build files should be included in the source repository unless there is a good reason.

And I don't think there is a downside of adding it to the ignore file. 
You are right that this should go into the ignore file which I just forgot to do so. : ) 
Maybe you can add it when you are removing the build folder?

Thanks!